src/metaswitch/sasclient/main.py

```diff
@@ -23,8 +32,9 @@ def __init__(self, system_name, system_type, resource_identifier, sas_address, s
         :param sas_address: The hostname or IP address of the SAS server to communicate with, (no
                             port)
         :param start: Whether the SAS client should start immediately
+        :param queue_length: The maximum number of messages to queue for sending to SAS
         """
-        self._queue = Queue.Queue()
+        self._queue = Queue.Queue(maxsize=queue_length)
```

Comment:
Is this measured in bytes or in number of messages?

Reply:
Messages - I'll add a comment.

Reply:
Actually, there is already a comment to that effect on the previous line (35) - I'm leaving it.

Reply:
Oh, didn't spot that - I stand corrected.